CorinneTest/IPersonFactory.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace CorinneTest
+{
+    interface IPersonFactory
+    {
+        Person CreatePerson(string myName, int myAge, Gender myGender, DateTime myBirthDate);
```

Comment:
this should return the interface `IPerson` to allow different implementations don't you think?

Reply:
yeah that would make sense for projects with different implementations. greg said i won't be doing that for this project so i'm deleting IPerson and IPersonFactory interfaces